doc/40_observatory/OBS_Augment_table_with_census.md

```diff
@@ -0,0 +1,125 @@
+
+## Name
+
+OBS_AUGMENT_TABLE_WITH_CENSUS - adds the requested dimension to the input table aggregating appropriately depending on the tables
+geometry type.
+
+## Synopsis
+
+```postgresql
+OBS_AUGMENT_TABLE_WITH_CENSUS (
+  target_table_name text,
+  column_name text
+);
+
+OBS_AUGMENT_TABLE_WITH_CENSUS (
+  target_table_name text,
+  column_name text
+);
+
+OBS_AUGMENT_TABLE_WITH_CENSUS (
+  target_table_name text,
+  column_name text,
+  time_span text DEFAULT '2009 - 2013',
+  geometry_level text DEFAULT '"us.census.tiger".block_group'
+);
+
+OBS_AUGMENT_TABLE_WITH_CENSUS (
```

Comment:
Yeah no problem. We wrote these earlier on, happy to not have them in the first release. These should be the only two functions that do any actual augmentation